### ISSUE TITLE
A logo, mascot, or other representation for the project

### ISSUE THREAD

**Issue by ELLIOTTCABLE** [melange-re/melange#135]
It's early days, but for a website, we'll need some sort of logo!

A lot of JavaScript projects use the 'boring square with two letters' design; personally, I'm more drawn to cute or lighthearted things.

I'm happy to personally throw some money at some wonderful artist to draw up some propositions for us, especially if some contributors can agree on a theme or character to supply them with.

(For what it's worth, I'd really like us to find a graphic designer who's 1. involved with the community a little bit, and 2. a PoC or other marginalized person. Again, just my 2¢.)

**Comment by EduardoRFS** [comment]
YES, something which is not boring would be really cool. @anmonteiro please this is an important issue

**Comment by coetry** [comment]
I agree with the more light heart vibe to keep pushing that fun hacker spirit. Would love some concepts similar to [Joe Camel](https://en.m.wikipedia.org/wiki/Joe_Camel)! 

![0E710F91-E03D-48F5-A684-5CFFF99249AF](https://user-images.githubusercontent.com/10926503/113534220-aec8e800-9595-11eb-953d-6b5a3bbe201e.jpeg)

**Comment by ELLIOTTCABLE** [comment]
We were just discussing this on Discord a tad; I was thinking perhaps a teddy bear? With lots of patches? Y'know, a cute … melange … of parts. :P

![image](https://user-images.githubusercontent.com/200/113760917-741d9780-96dc-11eb-8385-66fa488430ba.png)

**Comment by anmonteiro** [comment]
@ELLIOTTCABLE that does look cool, I like the color tone too. It'd be cool to get a sort of cartoon version of the patched teddy bear. what do folks think?

I can help with some $$$, no strong opinions on the artist.

**Comment by kennetpostigo** [comment]
I think that would be awesome! Also has there been any discussion about a potential website?

**Comment by anmonteiro** [comment]
website is being discussed at #105

**Comment by coproduto** [comment]
Based on the project name, may I suggest a [Shai-Hulud](https://en.wikipedia.org/wiki/Sandworm_(Dune))? :stuck_out_tongue:

**Comment by farukg** [comment]
Hi, i love this project cause i think its moving in the right direction. I had an idea for a logo based on the suggestions made here combining the patched up teddy and cafe melange. I have a very good friend @MarieOlligesgeers who was so kind to make this 😍:

![teddy-melange](https://user-images.githubusercontent.com/1577842/116865372-cc778480-ac09-11eb-9a71-5edf6e20c6bb.jpeg)

feel free to ask for changes, if you like :)

**Comment by fccm** [comment]
Edited with Gimp, converted to SVG with autotrace, and cleaned with Inkscape:
![melange-teddy-bear.svg](
http://decapode314.free.fr/re/melange-teddy-bear.svg)

**Comment by fccm** [comment]
Another idea:
![MeL-ange](http://decapode314.free.fr/re/MeL-ange.svg)

**Comment by fccm** [comment]
an attempt following the Shai-Hulud idea:

![shai-hulud2.svg](http://decapode314.free.fr/re/shai-hulud.svg)
[shai-hulud2.svg](http://decapode314.free.fr/re/shai-hulud.svg)

![shai-hulud2.svg](http://decapode314.free.fr/re/shai-hulud2.svg)
[shai-hulud2.svg](http://decapode314.free.fr/re/shai-hulud2.svg)

**Comment by fccm** [comment]
![MeL-ange.2.svg](http://decapode314.free.fr/re/MeL-ange.2.svg)
[MeL-ange.2.svg](http://decapode314.free.fr/re/MeL-ange.2.svg)
![MeL-ange.3.svg](http://decapode314.free.fr/re/MeL-ange.3.svg)
[MeL-ange.3.svg](http://decapode314.free.fr/re/MeL-ange.3.svg)
![MeL-ange.4.svg](http://decapode314.free.fr/re/MeL-ange.4.svg)
[MeL-ange.4.svg](http://decapode314.free.fr/re/MeL-ange.4.svg)

**Comment by fccm** [comment]
trying to make you change your mind about the square and letters:
![ML-melange-JS.svg](http://decapode314.free.fr/re/ML-melange-JS.svg)
[ML-melange-JS.svg](http://decapode314.free.fr/re/ML-melange-JS.svg)

**Comment by fccm** [comment]
for the sand-worm idea it would be fun to reuse the desert background of ocaml.org, and replace the camel by the sand-worm :)

**Comment by fccm** [comment]
![mel-ange.0.svg](http://decapode314.free.fr/re/mel-ange.0.svg)
[mel-ange.0.svg](http://decapode314.free.fr/re/mel-ange.0.svg)

![mel-ange.1.svg](http://decapode314.free.fr/re/mel-ange.1.svg)
[mel-ange.1.svg](http://decapode314.free.fr/re/mel-ange.1.svg)

![mel-ange.2.svg](http://decapode314.free.fr/re/mel-ange.2.svg)
[mel-ange.2.svg](http://decapode314.free.fr/re/mel-ange.2.svg)

![mel-ange.3.svg](http://decapode314.free.fr/re/mel-ange.3.svg)
[mel-ange.3.svg](http://decapode314.free.fr/re/mel-ange.3.svg)

**Comment by fccm** [comment]
How much should I give to play/use this project at home?
How much would help you working on this project?

**Comment by trite** [comment]
I have another direction to toss in for consideration. It might be too indirect, but though it could be fun to play on `Melange` being referred to as `Spice` in the Dune series, using a spice bottle and pile of spices. Something like:

![Melange-spice-bottle-logo-2](https://user-images.githubusercontent.com/5068415/200371023-4c241f0f-2e55-4f18-a608-1bcf4bb43e4d.svg)
Basic idea

![Melange-spice-bottle-logo-1](https://user-images.githubusercontent.com/5068415/200371026-c24a6e8e-892a-4e80-b49c-5c578d2ad4b7.svg)
Probably better spacing

![Melange-spice-logo-color-1](https://user-images.githubusercontent.com/5068415/200371018-106e23d3-a042-40e0-82c3-510f826cd0d6.svg)
Trying out some colors from the `Dune` logo

**Comment by ELLIOTTCABLE** [comment]
> ![](https://camo.githubusercontent.com/ec39fd20d91e35527d5f145f94e19f31a7284049e18287b13a80eebfb1075e5f/687474703a2f2f64656361706f64653331342e667265652e66722f72652f6d656c616e67652d74656464792d626561722e737667)

I think this came out pretty well!
